README.md

```diff
@@ -21,4 +21,112 @@ An acme client that integrates with Kubernetes to automatically generate and ren
 
 ## Usage
 
-* _coming soon_
+### Initial setup
+
+At the moment kube-acme is to some extent limited by namespaces, and aspecially the inability
+of ingress to reference service from different namespace. This means that for an ingress
+to be able to properly work, a "local" (as in the "same namespace") kube-acme service is required.
+Still, to properly establish locks, a dedicated "acme" namespace is expected to exist.
+
+```
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: acme
+```
+
+To connect to the acme server a registers user is required (while this will most likely be handled by `kube-acme` in future,
```

Comment:
or maybe "A user must be registered with the acme server in order to retrieve certificates, and it's credentials stored in a secret for kube-acme to read. Currently this is a manual process, although will most like be handled by kube-acme in future." or words to that affect :)